React/Views/RCTView.m

```diff
@@ -118,6 +118,11 @@ - (instancetype)initWithFrame:(CGRect)frame
     _hitTestEdgeInsets = UIEdgeInsetsZero;
 
     _backgroundColor = super.backgroundColor;
+
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(voiceOverStatusDidChange:)
+                                                 name:UIAccessibilityVoiceOverStatusChanged
+                                               object:nil];
```

Comment:
that's a great point, I'll investigate a solution that removes the dependency on NSNotificationCenter. Though, I can't imagine there would be thousands of views instantiated at one time in an application - if that were the case this one observer wouldn't be the issue, it would be the memory bloat from the views themselves.